### PR TITLE
Add ducc to the build procedure via rpm

### PR DESCRIPTION
**Pull request by siscia**
This PR add ducc to the standard building of cvmfs using the universal.spec

### CODE REVIEW

File: packaging/rpm/cvmfs-universal.spec [hunk 1]
```diff
@@ -35,7 +35,7 @@ Summary: CernVM File System
 Name: cvmfs
 Version: 2.6.0
 Release: 1%{?dist}
-Source0: https://ecsft.cern.ch/dist/cvmfs/%{name}-%{version}.tar.gz
+Source0: https://ecsft.cern.ch/dist/cvmfs/%{name}-%{version}/source.tar.gz
```
Comment (jblomer) [inline packaging/rpm/cvmfs-universal.spec]: The name should stay as before, i.e. https://ecsft.cern.ch/dist/cvmfs/%{name}-%{version}/%{name}-%{version}.tar.gz

File: packaging/rpm/cvmfs-universal.spec [hunk 2]
```diff
@@ -200,6 +200,13 @@ Group: Application/System
 %description unittests
 CernVM-FS unit tests binary.  This RPM is not required except for testing.
 
+%package ducc
+Summary: ducc: Daemon Unpacker Containers in CVMFS
```
Comment (jblomer) [inline packaging/rpm/cvmfs-universal.spec]: Daemon Unpacking?

File: packaging/rpm/cvmfs-universal.spec [hunk 3]
```diff
@@ -510,6 +519,9 @@ fi
 %{_bindir}/cvmfs_test_shrinkwrap
 %doc COPYING AUTHORS README.md ChangeLog
 
+%files ducc
+%{_bindir}/cvmfs_ducc
+
 %changelog
```
Comment (jblomer) [inline packaging/rpm/cvmfs-universal.spec]: You can add this change to the change log

File: packaging/rpm/cvmfs-universal.spec [hunk 4]
```diff
@@ -10,6 +10,8 @@
 %endif
 %if 0%{?el7} || 0%{?fedora}
 %define selinux_cvmfs_server 1
+%endif
+%if 0%{?el7} || 0%{?fc29}
```
Comment (jblomer) [inline packaging/rpm/cvmfs-universal.spec]: Is there a possibility to check for fedora >= 29?

Comment (siscia) [reply packaging/rpm/cvmfs-universal.spec]: On https://github.com/cvmfs/cvmfs/pull/2323/commits/7b6f09c62809de6883592b1defea63ea47105085

**Comment by siscia** [comment]
I believe this PR is ready.

A build produce by this PR is here: https://ecsft.cern.ch/dist/cvmfs/nightlies/cvmfs-git-934/

**Pull request merged**